clang/test/compiler-course/beskhmelnova_k_implicit_conversions/beskhmelnova_k_implicit_conversions_test.cpp

```diff
@@ -0,0 +1,20 @@
+// RUN: %clang_cc1 -load %llvmshlibdir/ClangAST_1_BeskhmelnovaK_FIIT1_ClangAST%pluginext -plugin ClangAST_1_BeskhmelnovaK_FIIT1_ClangAST -fsyntax-only %s 2>&1 | FileCheck --match-full-lines %s
+
+// CHECK Total implicit conversions: 5
+
+// Проверка порядка выводимых строк:
+// CHECK: Function sum
+// CHECK-NEXT: float -> double: 1
+// CHECK-NEXT: int -> float: 1
+// CHECK: Function mul
+// CHECK-NEXT: double -> int: 1
+// CHECK-NEXT: float -> double: 1
+// CHECK-NEXT: float -> int: 1
+
```

Comment:
Consider adding your own test cases. For example, test for classes that have conversion constructor:

```
class X {
   int x;
public:
   X(int);
}
```